pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/MedicalRecord.java

```diff
@@ -423,6 +437,13 @@ public static List<MedicalRecord> getInstitutions(@NonNull String realm, @NonNul
         return medicalRecords;
     }
 
+    public static MedicalRecord copy(int newOsteoInstitutionId, int newMedicalRecordId, MedicalRecord that) {
```

Comment:
is not implementing `Cloneable` interface with using `Object.clone()` method better alternative of copy?

Reply:
I think this method is dead code